packages/app-accounts/src/modals/Backup.tsx

```diff
@@ -78,14 +77,23 @@ class Backup extends React.PureComponent<Props, State> {
           {t('Backup account')}
         </Modal.Header>
         <Modal.Content className='app--account-Backup-content'>
-          <AddressSummary value={pair.address()} />
-          <Password
-            isError={!isPassValid}
-            label={t('unlock account using the password')}
-            onChange={this.onChangePass}
-            tabIndex={0}
-            value={password}
-          />
+          <AddressRow
+            isInline
+            value={pair.address()}
+          >
+            <p>{t('A encrypted backup file will be created in once you have pressed the "Download" button. This can be used to re-import your account on any other machine.')}</p>
```

Comment:
typo
```suggestion
            <p>{t('An encrypted backup file will be created once you have pressed the "Download" button. This can be used to re-import your account on any other machine.')}</p>
```